schematic/utils/io_utils.py

```diff
@@ -31,12 +31,12 @@ def export_json(json_doc, file_path):
 def load_default():
     """Load biolink vocabulary
     """
-    biothings_path = os.path.join(DATA_PATH, 'schema_org_schemas', 'biothings.jsonld')
+    biothings_path = os.path.join('data', 'schema_org_schemas', 'biothings.jsonld')
     return load_json(biothings_path)
 
 
 def load_schemaorg():
     """Load SchemOrg vocabulary
     """
-    schemaorg_path = os.path.join(DATA_PATH, 'schema_org_schemas', 'all_layer.jsonld')
+    schemaorg_path = os.path.join('data', 'schema_org_schemas', 'all_layer.jsonld')
```

Comment:
As far as I can tell, the `schema_org_schemas` was dropped from the `config.yml` file. I think this was slated for bundling with the package, but I don't see that implemented in this PR. I'm fine with a separate PR for bundling resource files like the schema.org JSON-LD file, but this script technically won't work until that's done. How should this be handled until then?